src/components/AuthContainer.js

```diff
@@ -14,11 +16,13 @@ import TermsOfService from 'src/pages/TermsOfService'
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
   const { isSignedIn, registrationStatus, acceptedTos, profile } = useStore(authStore)
+  const seenAzurePreview = useStore(azurePreviewStore) || false
```

Comment:
I wonder if the || false is necessary?

Reply:
I’m doing `=== false` elsewhere so wanted to init it to false if the store doesn’t exist yet